src/main/java/fi/foyt/foursquare/api/FoursquareApi.java

```diff
@@ -75,6 +75,12 @@
   private static final String DEFAULT_VERSION = "20140131";
 
   /**
+   * CDI eyes only
+   * @deprecated CDI eyes only
+   */
+  public FoursquareApi() {}
```

Comment:
Do we want to actually mark this as deprecated? If so, we should use the `@Deprecated` annotation in the actual source, not just the javadoc.

Also a personal preference to have at least a new line between braces